templates_jinja2/suggest_match_video_playlist.html

```diff
@@ -0,0 +1,101 @@
+{% extends "base.html" %}
+
+{% block title %}The Blue Alliance - Add Match Video{% endblock %}
+
+{% block meta_description %}Add match videos to The Blue Alliance.{% endblock %}
+
+{% block content %}
+<script type="text/javascript">
```

Comment:
Put the javascript in a {% block inline_javascript %}{% endblock %}

https://github.com/the-blue-alliance/the-blue-alliance/blob/master/templates_jinja2/base.html#L166